sst-containers/README.md

```diff
@@ -194,7 +194,8 @@ the HPC system.
 
 Note that the architecture of the desktop and HPC system should be compatible
 (e.g., both x86_64), otherwise you may need to adjust the build process to
-target the HPC system's architecture.
+target the HPC system's architecture. See the [Cross-Building for Different Architectures](#cross-building-for-different-architectures) section below for detailed instructions on
```

Comment:
Not sure it matters, but this line is long compared to the others in the same paragraph

Reply:
I am generally ignoring links that are long in documents because that road is dark, and filled with terror.